src/mds/MDSAuthCaps.cc

```diff
@@ -382,25 +382,29 @@ bool MDSAuthCaps::parse(string_view str, ostream *err)
   }
 }
 
-bool MDSAuthCaps::merge(MDSAuthCaps newcap)
+/* Check if the "cap grant" is already present in this cap object. If it is,
+ * return false. If not, add it and return true.
+ *
+ * ng = new grant, new mds cap grant, shorter name makes this code more
+ * succinct.
```

Comment:
really? Code is succinct when it's a) cleanly/clearly written and b) adequately commented. I disagree!!

Reply:
Ok. Removed.